src/utils/utils.js

```diff
@@ -12,10 +13,46 @@ export function handleImageClick(thisObj) {
   const title = thisObj.name;
   const link = thisObj.link;
 
-  cardPopup.open({ title, link });
+  previewPopup.open({ title, link });
 }
-
+/*
 export function createCard(item) {
   const card = new Card(item, "#card-template", handleImageClick);
   return card.generateCard();
 }
+*/
+export function createCard(item) {
+  const card = new Card(item, "#card-template", handleImageClick, {
+    handleDelete: (cardElement, id) => {
+      deletePopup.open();
+      deletePopup.catchSelectedCard(cardElement, id);
+    },
+    handleLike: (id) => {
+      api.addLike(id).then((r) => {
+        card._handleLikeButton();
```

Comment:
`_handleLikeButton`

Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.